src/WinUI/ArcGIS.WinUI.Viewer/Samples/Analysis/DistanceMeasurement/readme.metadata.json

```diff
@@ -14,7 +14,7 @@
     ],
     "offline_data": [],
     "redirect_from": [
-        "/net/latest/winui/sample-code/distance-measurement-analysis.htm"
```

Comment:
@williambohrmann3 For this one for example, since it was not consolidated, would we need both or just to restore the latest?

Reply:
both